continuous_integration/environment.yml

```diff
@@ -0,0 +1,13 @@
+name: test
+channels:
+  - conda-forge
+  - defaults
```

Comment:
Do we need `defaults` at all? If we need it, this is fine. Otherwise I'd prefer to remove to avoid issues in the future 

```suggestion
```

Reply:
It's not strictly needed, but it's the pattern we have everywhere else (dask, distributed). It's also what we already used until now (see `continuous_integration/condarc`)